### ISSUE TITLE
The results of SimCLR is not match with the original paper

### ISSUE THREAD

**Issue by HuangChiEn** [vturrisi/solo-learn#305]
At the first, thansk for your release of such amazing framework, which almost cover all of the SOTA-SSL methods.

While enabling the multi-GPU training (up to 4 gpus), the linear evaluation results do not match the SimCLR report in Table B.1 in appendixes. 
![image](https://user-images.githubusercontent.com/52521165/193165783-307a465d-5aa6-4fc4-bbfa-4dcbbbdf9bfd.png)
> By the way, I have rearranged the config file by [easy-configer](https://pypi.org/project/easy-configer/#description), so that it's easy to execute in basah and easy read in wandb.

The results in wandb based on solo-learn framework can be found in the following link :
https://wandb.ai/josef/solo-learn/runs/ah1hwzpm/overview
In short, we found that we got 60.83% (about 4% drop) of linear-eval Top-1 acc.

Any suggestion will be appreciated!!

**Comment by vturrisi** [comment]
Hi. Thanks for using our library. I would first check if all parameters match those in the SimCLR paper. Apart from that, I'm not sure if we have any issues with SimCLR, but I can eventually take a closer took at it.

**Comment by HuangChiEn** [comment]
> Hi. I took a brief look at your config, and your eta for Lars is wrong. The default values for Lars should parameters be:
> 
> kwargs:     clip_lr: False     eta: 0.001     exclude_bias_n_norm: True
> 
> They differ from those for Imaneget100 as we found that a larger eta benefits the models there. There might be other issues that I didn't spot as well.

Sorry, reply lately and thanks for your comment.
Yes, i think i will adjust the eta values according to your suggestion.
